src/Models/Classes/3DVConfig.ts

```diff
@@ -98,38 +93,8 @@ export const defaultOnClickPopover: IPopoverVisual = {
     }
 };
 
-export const defaultStatusColorVisual: IExpressionRangeVisual = {
-    type: VisualType.ExpressionRangeVisual,
-    expressionType: 'NumericRange',
-    valueExpression: '',
-    valueRanges: [],
-    objectIDs: {
-        expression: 'objectIDs'
-    }
-};
-
-export const defaultAlertVisual: IExpressionRangeVisual = {
-    type: VisualType.ExpressionRangeVisual,
-    expressionType: 'CategoricalValues',
-    valueExpression: '',
-    valueRanges: [
-        {
-            id: '',
-            values: [true],
-            visual: {
-                color: defaultSwatchColors[0].item,
-                iconName: defaultSwatchIcons[0].item,
-                labelExpression: ''
-            }
-        }
-    ],
-    objectIDs: {
-        expression: 'objectIDs'
-    }
-};
-
-export const getDefaultVisualRule = (): IExpressionRangeVisual => ({
-    id: createGUID(),
+export const getDefaultVisualRule = (id?: string): IExpressionRangeVisual => ({
+    id: id ? id : createGUID(),
```

Comment:
is there a reason why we pass id here as optional to get a default visual rule object? I see that you use it for passing mock_id but we could get a default visual rule as a variable and pass its guid generated id in the below state action instead. The reason is that the getDefault... methods should be similar by setting id to either empty string (to generate the guid later at the time of save for example) or a createGuid as we have been doing in that file (for consistency).

![image](https://user-images.githubusercontent.com/45217314/203185881-98f8fb64-d696-4e61-a6fb-c6ce2d350b60.png)